Monika After Story/game/script-farewells.rpy

```diff
@@ -345,6 +345,51 @@ label bye_prompt_sleep:
     $ persistent._mas_greeting_type = store.mas_greetings.TYPE_SLEEP
     return 'quit'
 
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_prompt_to_games",
+            unlocked=True,
+            prompt="I'll play other games.",
+            pool=True
+        ),
+        eventdb=evhand.farewell_database
+    )
+
+label bye_prompt_to_games:
+    m 1esa "Ok [player]!"
+    m 1eka "I wish I could come too and see your main character."
+    m 3rsc "But I mentioned, it depends on what you gonna play."
+    m 1eka "I couldn't stay in violance atmosphere."
```

Comment:
"I couldn't stay in a violent atmosphere."

Might be better to say "can't or "I'd rather not".